frontend/Readme.md

```diff
@@ -0,0 +1,42 @@
+## Service-Center UI
+
+Service-Center UI enables user to view the list of MicroServices registered in SC. Users can view the detailed information of their MicroServices, Instances and Schemas.
+Service-Center UI also offers a unique feature of testing the Schemas of their MicroServices from UI, Users can also download the html client for their Schemas.
+
+### QuickStart Guide
+
+Easiest way to get started with Service-Center UI is to download the release from [here](https://github.com/ServiceComb/service-center/releases) and then untar/unzip it based on your OS and run start.sh/start.bat.
+This will bring up the Service-Center UI on [http://127.0.0.1:30101](http://127.0.0.1:30101).
```

Comment:
Port 30101 is already listen by Config Center, so please change the default port.

Reply:
@little-cui updated the port to 30103